libraries/AP_Scripting/examples/DroneCAN_test.lua

```diff
@@ -0,0 +1,261 @@
+--[[
+    example of creating and sending DroneCAN messages
+--]]
+---@diagnostic disable: param-type-mismatch
```

Comment:
Cheating ;) What is the error? I have seen them with string unpack, in that case you can cast like this:

https://github.com/ArduPilot/ardupilot/blob/1431a08b8d5a8fcaf06f3a9320de5e8a8e81b737/libraries/AP_Scripting/examples/message_interval.lua#L48-L51

The goal is to ideally remove the disabling of the check. If the cast does not work then a per line disable rather than a per file would be preferred.

Reply:
Ah, the dreaded logger one. You should be able to swap to colon and have it pass. 
```
logger:write("ASPL", 'SP,DP,ST,DT,SAT,PIT', 'ffffff', ...
```

Reply:
changed